server/lib/tags.ts

```diff
@@ -0,0 +1,17 @@
+export const validateTags = (tags: string[]): void => {
+  if (tags) {
+    // Limit to max 30 tags
+    if (tags.length > 30) {
+      throw new Error(`Conversations cannot have more than 30 tags. Please remove ${30 - tags.length} tag(s).`);
```

Comment:
If it's used for something else that Conversations, the message is wrong.

Reply:
🎯